integrations/livepeer/components/form-livepeer-api-key.tsx

```diff
@@ -0,0 +1,97 @@
+'use client'
+
+import { useEffect, useState } from 'react'
+
+import { useForm } from 'react-hook-form'
+import { BiInfoCircle } from 'react-icons/bi'
+
+import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/ui/tooltip'
+import { useToast } from '@/lib/hooks/use-toast'
+
+import { FailedToFetchError, NotFoundError, PermissionError, useCheckLivepeerApiKey } from '../hooks/use-check-livepeer-api-key'
+import { useIsLivepeerApiKeySet, useLivepeerApiKey } from '../hooks/use-livepeer-api-key'
+
+interface livepeerForm {
+  apiKey: string
+}
+
+export function FormLivepeerApiKey() {
+  const { register, handleSubmit, watch } = useForm<livepeerForm>()
+  const { checkLivepeerApiKey } = useCheckLivepeerApiKey()
+  const { toast, dismiss } = useToast()
+  const [, setLivepeerApiKey] = useLivepeerApiKey()
+  const [isLoading, setIsLoading] = useState<boolean>(false)
```

Comment:
Adding the state declarations at the top of the component adds clarity imo

```suggestion
  const [isLoading, setIsLoading] = useState<boolean>(false)
  const { register, handleSubmit, watch } = useForm<livepeerForm>()
  const { checkLivepeerApiKey } = useCheckLivepeerApiKey()
  const { toast, dismiss } = useToast()
  const [, setLivepeerApiKey] = useLivepeerApiKey()
```